src/catalog/Cargo.toml

```diff
@@ -73,10 +73,13 @@ tokio-postgres = { version = "0.7.8" }
 [build-dependencies]
 anyhow = "1.0.66"
 md-5 = "0.10.5"
-protobuf-src = "1.1.0"
```

Comment:
Late to comment, but we should probably add a check to [ci/test/lint-deps.sh](https://github.com/MaterializeInc/materialize/blob/5aa09404319513a4a9ad913849c396af08907bc2/ci/test/lint-deps.sh) to make sure nobody introduces a dependency on `protobuf-src` again.

Reply:
Here's a diff
```diff
diff --git a/ci/test/lint-deps.sh b/ci/test/lint-deps.sh
index 85064519b4..365adbe01a 100755
--- a/ci/test/lint-deps.sh
+++ b/ci/test/lint-deps.sh
@@ -37,6 +37,7 @@ crates=(
     # be disabled using --no-default-features
     tikv_jemalloc_ctl
     tikv_jemallocator
+    protobuf_src
 )

 rewrite=false
(
```

Reply:
Great idea! Just put up https://github.com/MaterializeInc/materialize/pull/26666 to do this